ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Create a bootable SD card
+permalink: /documentation/ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md.html
+redirect_from: /documentation/ConsumerEdition/DragonBoard-410c/Guides/GPT.md/
```

Comment:
does that mean that the 'old' link will still work? if so, good! if not, then can you please make sure we have a link on the old page? i am sure i have shared this link many times..

Reply:
I guess the old link will redirect to new page. But I do need @shovanuk to confirm this. Also, one more thing, I have the old guide redirect to both the guides. So, don't know how this will work.

I mean, **redirect from: .GPT.md** is present for both guides. We should remove this from one of the guides. So, eventually one guide will become new guide which doesn't have any previous redirect.